/*
 * The MIT License
 *
 * Copyright 2016 CloudBees, Inc.
 *
 * Permission is hereby granted, free of charge, to any person obtaining a copy
 * of this software and associated documentation files (the "Software"), to deal
 * in the Software without restriction, including without limitation the rights
 * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
 * copies of the Software, and to permit persons to whom the Software is
 * furnished to do so, subject to the following conditions:
 *
 * The above copyright notice and this permission notice shall be included in
 * all copies or substantial portions of the Software.
 *
 * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
 * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
 * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
 * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
 * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
 * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
 * THE SOFTWARE.
 */

package test;

import org.junit.jupiter.api.Test;

public class SomeTest extends Base {

    @Test
    public void test1() {
        try {
            run();
        } catch (Exception e) {
            System.out.println("Exception in test1.");
        }
    }

    @Test public void test2() {
        try {
            run();
        } catch (Exception e) {
            System.out.println("Exception in test2.");
        }
    }

    @Test public void test3() {
        try {
            run();
        } catch (Exception e) {
            System.out.println("Exception in test3.");
        }
    }

    @Test public void test4() {
        try {
            run();
        } catch (Exception e) {
            System.out.println("Exception in test4.");
        }
    }

    @Test public void test5() {
        try {
            run();
        } catch (Exception e) {
            System.out.println("Exception in test5.");
        }
    }

    @Test public void test6() {
        try {
            run();
        } catch (Exception e) {
            System.out.println("Exception in test6.");
        }
    }

    @Test public void test7() {
        try {
            run(20.0);
        } catch (Exception e) {
            System.out.println("Exception in test7.");
        }
    }

    @Test public void test8() {
        try {
            run( 6);
        } catch (Exception e) {
            System.out.println("Exception in test8.");
        }
    }

    @Test public void test9() {
        try {
            run();
        } catch (Exception e) {
            System.out.println("Exception in test9.");
        }
    }

    @Test public void test10() {
        try {
            run(1.0);
        } catch (Exception e) {
            System.out.println("Exception in test10.");
        }
    }

    @Test public void test11() {
        try {
            run(0.5);
        } catch (Exception e) {
            System.out.println("Exception in test11.");
        }
    }

    @Test public void test12() {
        try {
            run(0.25);
        } catch (Exception e) {
            System.out.println("Exception in test12.");
        }
    }

}